opentimelineio_contrib/adapters/aaf_adapter/aaf_writer.py

```diff
@@ -316,24 +291,24 @@ def aaf_transition(self, otio_transition):
             "OperationGroup"
         ]["Operation"]["Name"]
 
+        # Create OperationDefinition
         op_def = self.aaf_file.create.OperationDef(uuid.UUID(effect_id), op_def_name)
         self.aaf_file.dictionary.register_def(op_def)
         op_def.media_kind = self.media_kind
```

Comment:
Can you remind me, when/where is self.media_kind set?

Reply:
It is a method overridden to return "picture" or "sound" like this:
https://github.com/PixarAnimationStudios/OpenTimelineIO/blob/2ebba59d872f60b65e434a5fa534674081faadf9/opentimelineio_contrib/adapters/aaf_adapter/aaf_writer.py#L385